linux/jdk/alpine/build.gradle

```diff
@@ -0,0 +1,143 @@
+plugins {
+	id "java"
+}
+
+ext {
+	junitVersion = "5.9.0"
+	testcontainersVersion = "1.17.3"
```

Comment:
1.17.5 now

Reply:
I'd prefer to not add dependency changes in this PR. We should add dependabot to handle the gradle updates